pkg/apis/devices/v1alpha2/device_instance_types.go

```diff
@@ -42,6 +42,11 @@ type DeviceSpec struct {
 	// Refer to k8s.io/kubernetes/pkg/apis/core NodeSelector for more details
 	// +optional
 	NodeSelector *v1.NodeSelector `json:"nodeSelector,omitempty"`
+	// NodeName is a request to schedule this device onto a specific node. If it is non-empty,
+	// the scheduler simply schedules this device onto that node, assuming that it fits
+	// resource requirements.
+	// +optional
+	NodeName string `json:"nodeName,omitempty"`
```

Comment:
I think we need 2 `NodeName`. One is in the field of `Spec` and it means users specify the node for device deployment, which could be allowed to be empty. The other one is in the field of `Status` and it means the device is scheduled to the node which could not be empty.